fm/echonest.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
I've created directory `thirdparty` in https://github.com/thisissoon/FM-API/pull/18 for all 3rd party libraries instead of keep them in the root directory..